mailpoet/RoboFile.php

```diff
@@ -448,12 +448,13 @@ public function doctrineGenerateCache() {
   }
 
   /**
-   * Creates a new migration file. Creates Db level migration. Use --level=App to create an application level migration.
-   * @param $opts array{level: string}
+   * Creates a new migration file. Use `migrations:new db` for a Db level migration or `migrations:new app` for App level migration.
+   * @param $level string - db or app
    */
-  public function migrationsNew($opts = ['level' => \MailPoet\Migrator\Repository::MIGRATIONS_LEVEL_DB]) {
+  public function migrationsNew($level) {
     $generator = new \MailPoet\Migrator\Repository();
-    $result = $generator->create($opts['level']);
+    $level = ucfirst(strtolower($level));
```

Comment:
This is just a super tiny unimportant idea, but maybe I'd internally work with the lowercased slugs (`app` and `db`) and put the `ucfirst` only to the namespace and filename constructing methods. It would then display in outputs and logs also as `db` and `app`, which are the easiest to use on the CLI and it would me more consistent with the `status` field:

<img width="444" alt="Screenshot 2023-07-25 at 10 06 13" src="https://github.com/mailpoet/mailpoet/assets/141436/5a27ea7f-5c84-4c4f-a567-112d5fbc965e">

Reply:
I like the idea. I went with the first capital letter so that the constants match directory names, and we name directories with CamelCase. I'll use small letters in the constants and ucfirst for the namespace and directory.